web/src/components/SkipLinks.tsx

```diff
@@ -0,0 +1,53 @@
+import KeyboardReturnIcon from '@mui/icons-material/KeyboardReturn'
+import Backdrop from '@mui/material/Backdrop'
+import Button from '@mui/material/Button'
+import Chip from '@mui/material/Chip'
+import { styled } from '@mui/material/styles'
+import React, { ReactElement, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import { MAIN_ELEMENT_ID } from '../constants'
+
+const SkipLink = styled(Button)`
+  position: fixed;
+  display: flex;
+  left: 20px;
+  top: 50px;
+  gap: 8px;
+  background-color: ${props => props.theme.palette.background.paper};
+  opacity: 0;
```

Comment:
🤔 Do we need the opacity? I don't see a difference without setting the opacity at all.